src/cc/bcc_elf.c

```diff
@@ -196,20 +201,254 @@ static int listsymbols(Elf *e, bcc_elf_symcb callback, void *payload) {
   return 0;
 }
 
-int bcc_elf_foreach_sym(const char *path, bcc_elf_symcb callback,
-                        void *payload) {
+static Elf_Data * get_section_elf_data(Elf *e, const char *section_name) {
+	Elf_Scn *section = NULL;
+	GElf_Shdr header;
+	char *name;
```

Comment:
Absolutely. That's my punishment for making these edits on a box without my usual .vimrc :)